README.md

```diff
@@ -191,9 +191,46 @@ yarn test
 You can also run the entire `zk-groups` using docker by running below command in the project root:
 
 ```sh
-docker-compose up -d # or docker compose up -d
+docker-compose up -d
 ```
 
+### Local Development
+
+You can start dependencies essential for local development like a local ethereum network (with data persistance) and TheGraph node using the below command:
+
+```sh
+docker-compose -f docker-compose.dev.yml up -d
+```
+
+The ethereum node (ganache) started on port `8545` will have the following accounts pre-funded with 100 ETH:
+
+```sh
+0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1
+Private Key: 0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d
+```
+
+This is the same first account generated by hardhat network as well.
```

Comment:
The first Hardhat account should be this one: 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266, isn't it different?

Reply:
Yea, my bad. This is the first of ganache. I have updated the addresses.